src/monodroid/jni/debug-app-helper.cc

```diff
@@ -97,7 +109,7 @@ Java_mono_android_DebugRuntime_init (JNIEnv *env, jclass klass, jobjectArray run
 	}
 }
 
-#if defined (ANDROID) && !defined (RELEASE)
```

Comment:
I'm not sure I'm comfortable with defining `copy_file_to_internal_location()` in RELEASE runtimes.  This feels like a potential security hole for Play Store apps.

Reply:
This helper is never [built](https://github.com/xamarin/xamarin-android/blob/master/src/monodroid/CMakeLists.txt#L273-L287) or [used](https://github.com/xamarin/xamarin-android/blob/master/src/java-runtime/java/mono/android/MonoPackageManager.java#L73-L78) in [RELEASE](https://github.com/xamarin/xamarin-android/blob/master/src/java-runtime/java/mono/android/release/BuildConfig.java#L4) builds.